golem-cli/src/init.rs

```diff
@@ -193,14 +239,24 @@ fn validate_profile_override(
 
 #[derive(Debug, Clone, Copy, EnumIter)]
 enum ProfileType {
-    Golem,
+    GolemDefault,
+    GolemCustom,
     GolemCloud,
 }
 
 impl Display for ProfileType {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         match self {
-            ProfileType::Golem => write!(f, "{}. For stand-alone installations", "Golem".bold()),
+            ProfileType::GolemDefault => write!(
+                f,
+                "{}. For default docker compose configuration",
```

Comment:
`default docker` , should dot we call it just `default` ?

Reply:
It is not `default docker` it is `default "docker compose"`

Reply:
i.e. is don the default CLI confiduration. it is configuration for the default docker compose file